src/Plugin/os2web/DataLookup/ServiceplatformenBase.php

```diff
@@ -321,9 +373,36 @@ protected function query(string $method, array $request): array {
         'status' => FALSE,
         'error' => $e->faultstring,
       ];
+    } finally {
+      // Remove temporary certificate file.
+      if (file_exists($localCertPath)) {
```

Comment:
`$localCertPath` can be undefined here, might throw a warning

Reply:
Fixed in https://github.com/OS2web/os2web_datalookup/pull/13/commits/be58b709421feb73b532633dbf574234c6ff7c5f.